apify-docs-theme/src/config.js

```diff
@@ -10,6 +10,11 @@ if (process.env.LOCALHOST) {
     absoluteUrl = process.env.APIFY_DOCS_ABSOLUTE_URL;
 }
 
+let noIndex = false;
+if (process.env.NO_INDEX) {
+    noIndex = [true, 'true', 1, '1'].includes(process.env.NO_INDEX);
```

Comment:
nit: i dont think you can get anything else than a string out of `process.env` vars

<img width="250" alt="image" src="https://github.com/user-attachments/assets/56666a84-86ab-49fa-8140-79cfa20579f1" />

also the `if` block seems unnecessary, `const noIndex = ['true', '1'].includes(process.env.NO_INDEX ?? '');` will do the same on a single line (which can be inlined to the place where its used)

Reply:
Good point, will clean up